packages/twenty-front/src/modules/prefetch/components/PrefetchRunQueriesEffect.tsx

```diff
@@ -24,11 +24,20 @@ export const PrefetchRunQueriesEffect = () => {
       prefetchKey: PrefetchKey.AllFavorites,
     });
 
+  const { objectMetadataItems } = useObjectMetadataItems();
```

Comment:
refactor PrefetchQueries to make sure that they are leveraging operationSignatures